include/dlaf/executors.h

```diff
@@ -108,8 +75,15 @@ inline auto getHpExecutor<Backend::GPU>() {
 ///
 /// @tparam S source device.
 /// @tparam D destination device.
-template <Device S, Device D>
+template <Device S, Device D, typename Dummy = void>
 decltype(auto) getCopyExecutor() {
-  return internal::GetCopyExecutor<S, D>::call();
+  if constexpr (S == Device::CPU && D == Device::CPU) {
+    return pika::execution::parallel_executor{&pika::resource::get_thread_pool("default"),
+                                              pika::threads::thread_priority::normal};
+  }
+  else {
+    DLAF_STATIC_FAIL(Dummy,
+                     "Do not use getCopyExecutor for copying anymore. Prefer sender adaptors with CopyBackend.");
+  }
```

Comment:
This is due for removal but there is still one CPU-CPU copy that uses the executor rather than the sender adaptors. I've disallowed GPU copies with this one to discourage anyone from even thinking about using this anymore. This should be completely removed in #475.